synapse/lib/stormhttp.py

```diff
@@ -505,6 +505,8 @@ class HttpResp(s_stormtypes.Prim):
         {'name': 'reason', 'desc': 'The reason phrase for the HTTP status code.', 'type': 'str'},
         {'name': 'body', 'desc': 'The raw HTTP response body as bytes.', 'type': 'bytes', },
         {'name': 'headers', 'type': 'dict', 'desc': 'The HTTP Response headers.'},
+        {'name': 'url', 'type': 'str',
+         'desc': '''The response url. If the request was redirected, this would be the final URL in the redirection chain. If the status code is -1, then this is the request URL.'''},
```

Comment:
```suggestion
         'desc': 'The response URL. If the request was redirected, this would be the final URL in the redirection chain. If the status code is -1, then this is the request URL.'},
```